### ISSUE TITLE
Look into signature options

### ISSUE THREAD

**Issue by MarkDavidson** [TAXIIProject/TAXII-Specifications#47]
At a minimum, we should look into using the new version of XML DSIG (v1.1).

**Comment by aboese** [comment]
Agreed. Notably in the case of xmlsec, it appears from what I've seen that the standard is already largely supported.

SignXML looks like a better library, if you can choose it.
